Tests/bip32-ed25519-swiftTests/TestUtils.swift

```diff
@@ -1,7 +1,51 @@
+/*
+ * Copyright (c) Algorand Foundation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 import Foundation
+import Sodium
+import Clibsodium
 import Base32
 
 public struct TestUtils {
+    public static func cryptoSecretBoxEasy(cleartext: String, nonce: Data, sharedSecret: Data) -> Data {
```

Comment:
in this context it doesn't have to be a shared secret

Reply:
Changed it to symmetricKey (encryption and decryption key are symmetric).